### DISCUSSION TITLE
Log ingestion: how to filter out unwanted noise?

### DISCUSSION THREAD

**Discussion by bezio** [SigNoz/signoz#6218] [Q&A] [answered]
Hey everyone!

We're about to evaluate signoz and I am yet stunned about the easy way of getting it up and showing first results.
But I do miss something: is there any way to filter out (during ingestion of e.g. logs) data? 
In my actual "situation" we get approx 30% deprecation messages (php/symfony) of all logs. And I do not even want to ingest them... So am searching for some "Drop" or "Suppress" filter (e.g. pipeline?)

Thanks for any hint,
Ingo

**Comment by srikanthccv** [comment, marked as answer]
https://signoz.io/docs/logs-management/guides/drop-logs/

**Comment by bezio** [comment]
Thanks - looks like I missed that in the docs 🤦
